### PR TITLE
fix: updated test command so Circle will test properly

### PR DESCRIPTION
**Pull request by aviadhahami**
- [X] Ready for review

#### What does this PR do?
Updated the CircleCi step where it invoked `test` incorrectly;

[Slack thread](https://snyk.slack.com/archives/CFTAUCYQ7/p1589208414100000)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -143,7 +143,7 @@ jobs:
           command: npm run snyk-auth
       - run:
           name: Run tests
-          command: npm run test:test
+          command: npm test
```
Comment (anthogez) [inline .circleci/config.yml:146]: I think it should be `npm run test` 🤔

Comment (aviadhahami) [reply .circleci/config.yml:146]: it's the same, `npm t` === `npm test` === `npm run test`
just like `npm i`

Comment (aviadhahami) [reply .circleci/config.yml:146]: https://docs.npmjs.com/cli/test

Comment (anthogez) [reply .circleci/config.yml:146]: nvm what I said.

**Pull request merged**